lockbox-ios/Common/AppDelegate.swift

```diff
@@ -107,7 +107,8 @@ extension AppDelegate {
         } else if previous > current {
             //this would mean a user had an upgraded test version and has now downgraded to a previous version
             //we should wipe keychain data to remove any possible abnormalities
-            KeychainWrapper.wipeKeychain()
+            KeychainWrapper.sharedAppContainerKeychain.clearAllValues(for: .account)
+            KeychainWrapper.sharedAppContainerKeychain.clearAllValues(for: .database)
```

Comment:
In time, will we move back to `wipeKeychain()`?

Reply:
Yes. I like being explicit about the values for now, but I think in the future we could do this.